boards/common/stm32f103c8/Makefile.include

```diff
@@ -23,14 +23,17 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 # (ground) GPIO B1.
 ifeq ($(PROGRAMMER),dfu-util)
   export ROM_OFFSET ?= 0x2000 # Skip the space needed by the embedded bootloader
-  export FLASHER = dfu-util
-  export DEBUGGER = # no debugger
-  export RESET = # dfu-util has no support for resetting the device
+  # flash the board using DFU
+  # Set ALT to 2
+  DFU_ALT = 2
```

Comment:
These one should I think be replaced by `DFU_ALT ?= 2` and same for the `DFU_ID ?= value` in general.
Also here the comment before is quite not necessary in this state.

Reply:
`DFU_ALT` cannot be different for flashing purposes. The DFU bootloader can be different for every device, thus implementing alt modes depend on how the address mapping is done by the bootloader.

The `DFU_ALT` values given here for all devices correspond to the address mapped for the internal available flash.